src/windows/views/emojis_listview.py

```diff
@@ -159,12 +157,25 @@ def __init__(self, model):
         self.setWordWrap(False)
         self.setStyleSheet('QListView::item { padding-top: 2px; }')
 
+        # Get default emoji filter group
+        s = get_settings()
+        default_type = s.get('emoji_group_filter') or 'smileys-emotion'
+
         # Load initial emoji model data
         self.emojis_model.update_model()
-        self.refresh_view()
 
         # setup filter events
-        app = get_app()
-        app.window.emojisFilter.textChanged.connect(self.filter_changed)
-        if not app.window.mode == "unittest":
-            app.window.emojiFilterGroup.currentIndexChanged.connect(self.filter_changed)
+        self.win.emojisFilter.textChanged.connect(self.filter_changed)
+        if not self.win.mode == "unittest":
+            self.win.emojiFilterGroup.currentIndexChanged.connect(self.group_changed)
+
+        # Loop through emoji groups, and populate emoji filter drop-down
+        self.win.emojiFilterGroup.clear()
+        self.win.emojiFilterGroup.addItem(_("Show All"), "")
+        dropdown_index = -1
+        for index, emoji_type in enumerate(sorted(self.emojis_model.emoji_groups)):
+            self.win.emojiFilterGroup.addItem(_(emoji_type.capitalize()), emoji_type)
+            if emoji_type == default_type:
+                # Initialize emoji filter group to settings
+                dropdown_index = index
+        self.win.emojiFilterGroup.setCurrentIndex(dropdown_index)
```

Comment:
Like I think I mentioned earlier, doing this last ­— and in particular, _after_ the signal is connected — means that it takes care of refreshing the view once it's done.